middle_end/flambda2/terms/flambda_primitive.ml

```diff
@@ -613,15 +615,19 @@ let result_kind_of_nullary_primitive p : result_kind =
 
 let effects_and_coeffects_of_nullary_primitive p =
```

Comment:
Might be worth trying to add a return type annotation here.  Not sure if it will allow the `Effects.`, `Coeffects.` and `Placement.` qualifications to be removed (which would help legibility), but maybe.

Reply:
It does indeed allow to remove the qualifications, so I've done the change for the `effects_and_coeffects_of_*` functions for which it made sense.